[BUG] False positive on

**Describe the bug**

- Template expressions are considered in HTML attribute duplicates

```
                <input
                    name="{{name}}"
                    type="text"
                    autocomplete="off"
                    autocorrect="off"
                    class="form-control big-o-input-input {{#parse_error}}has-validation is-invalid{{/parse_error}}"
                    {{#parse_error}}aria-invalid="true"{{/parse_error}}
                    size="{{size}}"
                    {{^editable}}disabled{{/editable}}
                    {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}}
                    placeholder="{{placeholder}}"
                    {{#suffix}}aria-describedby="pl-big-o-input-{{uuid}}-suffix"{{/suffix}}
                    {{#aria_label}}aria-label="{{aria_label}}"{{/aria_label}}
                    {{^aria_label}}aria-labelledby="pl-big-o-input-{{uuid}}-label"{{/aria_label}}
                />
```

```
   18:83  error  The attribute '{{/aria_label}}' is duplicated  @html-eslint/no-duplicate-attrs
```
**To Reproduce**

```
  {
    files: ['file.mustache'],
    rules: {
      '@html-eslint/no-duplicate-attrs': 'error',
    },
    languageOptions: {
      parser: htmlParser,
      parserOptions: {
        templateEngineSyntax: htmlParser.TEMPLATE_ENGINE_SYNTAX.HANDLEBAR,
      },
    },
  },
```

**Expected behavior**

That the closing handlebar isn't considered an attribute.

**Additional context**
Add any other context about the problem here.

--- a/packages/eslint-plugin/lib/rules/no-duplicate-attrs.js
+++ b/packages/eslint-plugin/lib/rules/no-duplicate-attrs.js
@@ -62,9 +62,6 @@ module.exports = {
       if (Array.isArray(node.attributes)) {
         const attrsSet = new Set();
         node.attributes.forEach((attr) => {
-          if (!attr.key) {
-            return;
-          }
           if (hasTemplate(attr.key)) {
             return;
           }

--- a/packages/eslint-plugin/lib/rules/no-duplicate-attrs.js
+++ b/packages/eslint-plugin/lib/rules/no-duplicate-attrs.js
@@ -6,6 +6,7 @@
 const { RULE_CATEGORY } = require("../constants");
 const { createVisitors } = require("./utils/visitors");
 const { getRuleUrl } = require("./utils/rule");
+const { hasTemplate } = require("./utils/node");
 
 const MESSAGE_IDS = {
   DUPLICATE_ATTRS: "duplicateAttrs",
@@ -61,7 +62,13 @@ module.exports = {
       if (Array.isArray(node.attributes)) {
         const attrsSet = new Set();
         node.attributes.forEach((attr) => {
-          if (attr.key && attrsSet.has(attr.key.value.toLowerCase())) {
+          if (!attr.key) {
+            return;
+          }
+          if (hasTemplate(attr.key)) {
+            return;
+          }
+          if (attrsSet.has(attr.key.value.toLowerCase())) {
             context.report({
               node: attr,
               data: {

--- a/packages/eslint-plugin/tests/rules/no-duplicate-attrs.test.js
+++ b/packages/eslint-plugin/tests/rules/no-duplicate-attrs.test.js
@@ -29,6 +29,14 @@ ruleTester.run("no-duplicate-attrs", rule, {
         .split("\n")
         .join("\r\n"),
     },
+    {
+      code: `
+      <input
+    {{#aria_label}}aria-label="{{aria_label}}"{{/aria_label}}
+    {{^aria_label}}aria-labelledby="pl-big-o-input-{{uuid}}-label"{{/aria_label}}
+/>
+      `,
+    },
   ],
   invalid: [
     {

--- a/packages/eslint-plugin/tests/rules/no-duplicate-attrs.test.js
+++ b/packages/eslint-plugin/tests/rules/no-duplicate-attrs.test.js
@@ -36,6 +36,13 @@ ruleTester.run("no-duplicate-attrs", rule, {
     {{^aria_label}}aria-labelledby="pl-big-o-input-{{uuid}}-label"{{/aria_label}}
 />
       `,
+      languageOptions: {
+        parserOptions: {
+          templateEngineSyntax: {
+            "{{": "}}",
+          },
+        },
+      },
     },
   ],
   invalid: [
@@ -141,6 +148,30 @@ ruleTester.run("no-duplicate-attrs", rule, {
      
  ></div>
       `,
+            },
+          ],
+        },
+      ],
+    },
+    {
+      code: `<div foo foo {{aa}} {{aa}}> </div>`,
+      languageOptions: {
+        parserOptions: {
+          templateEngineSyntax: {
+            "{{": "}}",
+          },
+        },
+      },
+      errors: [
+        {
+          message: "The attribute 'foo' is duplicated.",
+          suggestions: [
+            {
+              messageId: "removeAttr",
+              data: {
+                attrName: "foo",
+              },
+              output: `<div foo  {{aa}} {{aa}}> </div>`,
             },
           ],
         },

